modules/check_in/app/services/travel_claim/configuration.rb

```diff
@@ -46,8 +46,8 @@ def service_name
     #
     # @return [Faraday::Connection] Configured HTTP connection
     #
-    def connection
-      Faraday.new(base_path, headers: base_request_headers, request: request_options) do |conn|
+    def connection(server_url: base_path)
+      Faraday.new(url: server_url, headers: base_request_headers, request: request_options) do |conn|
```

Comment:
updated to take optional url so that the initial auth request for the veis token can utilize the established pattern in the client